css/decanter.css

```diff
@@ -1702,11 +1715,110 @@ button,
   border-left: 1px solid #b6b1a9;
   border-right: 1px solid #b6b1a9; }
 
+html {
```

Comment:
Thanks for bringing this up, Shea. I think this does warrant further discussion, actually. I've played around with this a little and it doesn't appear that `html` needs to have the 100% width applied in order for this to work. So, that alone might address half of this.

If we do keep this implementation, I wonder if a helper mixin would be better, so that it's not compiled into the `decanter.css` file, but rather can be something that can be used and applied as needed (as alluded to, this would need good documentation).

There are also other options that we can explore that might not require us to manipulate HTML tags: https://css-tricks.com/couple-takes-sticky-footer

In either case, though, it seems that the larger discussion will/should center around if we want this functionality to live in Decanter or at our respective theme levels.

Reply:
I agree that decanter should not actually apply any styling to any element, especially not to `<html>` or `<body>`. I just want mixins that I can apply to the relevant container that allow me to stick the footer to the bottom of the page.